### ISSUE TITLE
List of Warnings

### ISSUE THREAD

**Issue by Lars-Kraemer** [MIC-DKFZ/patchly#1]
Hey @Karol-G,

I have a list of warnings that occur when using patchly.

/home/l727r/anaconda3/envs/sem_seg/lib/python3.10/site-packages/patchly/array_like.py:23: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
    data = torch.tensor(data)

These two are probably caused when patch_size==img_size

/home/l727r/anaconda3/envs/sem_seg/lib/python3.10/site-packages/patchly/sampler.py:392: RuntimeWarning: divide by zero encountered in scalar floor_divide
  additional_offset_s = [axis_squeeze_s[axis] // (len(axis_positions_s[axis]) - 1) for axis in range(n_axis_s)]

/home/l727r/anaconda3/envs/sem_seg/lib/python3.10/site-packages/patchly/sampler.py:393: RuntimeWarning: divide by zero encountered in scalar remainder
  remainder_offset_s = [axis_squeeze_s[axis] % (len(axis_positions_s[axis]) - 1) for axis in range(n_axis_s)]

Best,
Lars
